# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/IonBuilder.h
# Commit: 93842c3971f8
# Full Hash: 93842c3971f8e612537cba5d6d25b482c0d67a66
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-12-15 21:49:48
# Regressor Bug: 1601599
# File Overlap Count: 1
# Description:
#   Bug 1601599 part 3 - Simplify loop phi code in IonBuilder. r=tcampbell
#   
#   Before this bug:
#   
#   * We did not add phis for expression stack slots that aren't part of the loop.
# ==============================================================================

diff -r 4903b1b93526 -r 93842c3971f8 js/src/jit/IonBuilder.h
--- a/js/src/jit/IonBuilder.h	Sun Dec 15 11:39:00 2019 +0000
+++ b/js/src/jit/IonBuilder.h	Sun Dec 15 11:33:36 2019 +0000
@@ -234,9 +234,7 @@
   AbortReasonOr<MBasicBlock*> newOsrPreheader(MBasicBlock* header,
                                               jsbytecode* loopHead);
   AbortReasonOr<MBasicBlock*> newPendingLoopHeader(MBasicBlock* predecessor,
-                                                   jsbytecode* pc, bool osr,
-                                                   bool canOsr,
-                                                   unsigned stackPhiCount);
+                                                   jsbytecode* pc, bool osr);
 
   AbortReasonOr<MBasicBlock*> newBlock(MBasicBlock* predecessor,
                                        jsbytecode* pc) {